src/abstract_penalty.jl

```diff
@@ -1,12 +1,12 @@
 # Code based on code from OnlineStats by Josh day (see LICENSE.md)
 
-abstract Penalty
+abstract ParamCost
```

Comment:
I think cost makes more sense.  Loss refers specifically to the prediction error.  Cost includes prediction loss and any other penalties you may choose to add (in this case, a penalty due to your parameter values).  Here's a reference point to consider: http://stats.stackexchange.com/a/179027